src/components/SearchWidget.tsx

```diff
@@ -19,7 +19,7 @@ export function SearchWidget (props: SearchWidgetProps) {
                         ...props.searchParameters,
                         searchAddress: e.target.value
                     })
-                }} placeholder=" Defaults to 528 Kent St, Sydney, NSW, 2000" id="propertyAddress" required tabIndex={1} />
+                }} placeholder="Defaults to 528 Kent St, Sydney, NSW, 2000" id="propertyAddress" required tabIndex={1} />
```

Comment:
can we get rid of the placeholder here? The default value is 528 kent st. No placeholder is needed

Reply:
Why isn't a placeholder needed? Don't want the user to know what the default value is?

Reply:
won't the default value show?

Reply:
instead of setting placeholder, you could do `value={props.searchParameters.searchAddress}`

Reply:
It was changed to placeholder based on Chris's requests, the client didn't want to delete the value and wanted everything to be the placeholder grey

Reply:
Well, in that case I'll change it back to the placeholder!